modules/ivc_champva/spec/fixtures/form_json/vha_10_7959c.json

```diff
@@ -40,7 +40,7 @@
   "applicant_primary_through_employer": false,
   "applicant_primary_has_prescription": true,
   "applicant_primary_eob": "noEob",
-  "applicant_primary_insurance_type": { "hmo": true, "medigap": true },
+  "applicant_primary_insurance_type": "medigap",
```

Comment:
Updated to reflect that it's a radio not a checkbox so it will only ever have one value.